nova/nova/models.py

```diff
@@ -68,6 +77,8 @@ class TradingEquipment(models.Model):
 
     last_updated_current = models.DateTimeField(null=True, blank=True)
 
+    appeared_portfolios = models.ManyToManyField(Portfolio, related_name='equipments', blank=True)
```

Comment:
@furkanaydar 
I didn't get exactly what this is for. Right now, it seems that we are just modifying this but never reading this.

Reply:
We do actually. This field is used to establish a many to many relation between Portfolio and TradingEquipment objects. There is no such field in Portfolio objects and storing this connection as an array would be a bad choice.
In portfolio_view.py inside the portfolio_update function, we use this field to insert equipment objects to the Portfolio objects or remove them:

> equipment.appeared_portfolios.add(portfolio)
> equipment.appeared_portfolios.remove(portfolio)

Another use of this field would be a recommendation system. We can get the equipments with the most appearances in portfolios and recommend them to the users.